pkg/storage/cassandra_dependencies_test.go

```diff
@@ -57,6 +57,21 @@ func TestCassandraCustomTraceTTLParseError(t *testing.T) {
 	assert.Equal(t, "172800", foundValue, "unexpected TRACE_TTL environment var value")
 }
 
+func TestCassandraDefaultPort(t *testing.T) {
+	jaeger := v1.NewJaeger(types.NamespacedName{Name: "my-instance"})
+
+	b := cassandraDeps(jaeger)
+	assert.Len(t, b, 1)
+	assert.Len(t, b[0].Spec.Template.Spec.Containers, 1)
+	for _, e := range b[0].Spec.Template.Spec.Containers[0].Env {
+		if e.Name == "CQLSH_PORT" {
+			foundValue := e.Value
+			assert.Equal(t, "9042", foundValue, "unexpected CQLSH_PORT environment var value")
```

Comment:
It's missing the return statement, so that the test is now failing :-) You might also not need the `foundValue` anymore: you can use `e.Value` directly in the assertion.